pytest_fxa/plugin.py

```diff
@@ -27,10 +37,15 @@ def fxa_urls(pytestconfig):
     return ENVIRONMENT_URLS[os.getenv('FXA_ENV', 'stage')]
 
 
+@pytest.fixture(scope='session')
```

Comment:
This would use the same email address for every test method. I think this fixture should be method scoped, which would allow users to increase the scope if necessary.